### DISCUSSION TITLE
appear char issue

### DISCUSSION THREAD

**Discussion by yuusukealmal** [JanDeDobbeleer/oh-my-posh#5504] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

i use blue-owl with `MesloLGLDZ Nerd Font`
and it has a char that can be appear

![image](https://github.com/user-attachments/assets/570c373c-6de9-42c4-97e5-1b120023d12f)

### Theme

blue-owl

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Print the prompt in debug mode.

Usage:
  oh-my-posh prompt debug [bash|zsh|fish|powershell|pwsh|cmd|nu|tcsh|elvish|xonsh] [flags]

Flags:
  -h, --help           help for debug
  -p, --plain          plain text output (no ANSI)
      --pwd string     current working directory
      --shell string   the shell to print for

Global Flags:
  -c, --config string   config file path
```

**Comment by JanDeDobbeleer** [comment]
@yuusukealmal 2 questions, do you have a V3 Nerd Font, and, is this issue also present when forcing PowerShell to UTF-8?

**Reply by yuusukealmal** [reply]
i type `chcp` command and the result is `65001`, i think its a UTF-8, and for the first answer, my Nerd Font version is 1.210, i'm not sure if it is called a V3 though.
![image](https://github.com/user-attachments/assets/e05ea961-e0c5-424f-929b-7c469f241824)
